float64[] position_2
bool isReady_2
